### ISSUE TITLE
Very small, insignificant bug, but On the project and form permission pages, you can "add" the root admin to the permissions group, however, when you do so, nothing happens.

### ISSUE THREAD

**Issue by fritosxii** [matrix-msu/Kora3#42]
I assume this is because they're already added since they are root. IF this is the case, lets discuss, and either remove the admin from the list of users to add, or something.

**Comment by suuuth** [comment]
The code rendering the list of users:
```
@foreach($all_users as $user)
  <option value="{{$user->id}}">{{$user->first_name}} {{$user->last_name}}</option>
@endforeach
```
and root_person is the first person in $all_users

**Comment by suuuth** [comment]
changed to this
```
@foreach($all_users as $user)
    @if ($user->id !== 1)
        <option value="{{$user->id}}">{{$user->first_name}} {{$user->last_name}}</option>
    @endif
@endforeach
```

**Comment by austintruchan** [comment]
As far as I can tell, I can no longer see root as an option, so I say this is good.

**Comment by austintruchan** [comment]
In relation to this issue, I have found that a kora admin can add themselve as the project admin for the project they create. This then creates two instances of that admin, in the admin permissions for the project. 

- [ ] Admins creating a project should be removed from the list of admins to add to the project. 

- [ ] Additional information will go below the project admin select https://projects.invisionapp.com/d/main#/console/11107725/235009883/preview

- [ ] Same will need to happen for people who create a form, their names should be removed from the additional form admins select.

- [ ] Additional text will be added below the form admin select: https://projects.invisionapp.com/d/main#/console/11107725/236755359/preview

**Comment by suuuth** [comment]
ranch: `suth-42`

**Comment by austintruchan** [comment]
Hmmm. As an admin, I can still add myself as a project admin when creating a new project, and have my name appear twice in the project permissions. Ideally I wont be able to even see my name as an admin, so i can't add myself as the project admin twice.

**Comment by suuuth** [comment]
updated

**Comment by austintruchan** [comment]
is there anything special I need to do to see this? I am still seeing admin names on the create new project page.

**Comment by suuuth** [comment]
No, this update probably just isn't the fix you were looking for.  This should have removed the currently logged in user from the list, but I re-added the admin to the list.  Will update again

You want both the admin as well as the logged in user to not be seen in the list then, right?

**Comment by austintruchan** [comment]
Other admin names can stay actually, I was still seeing my own logged in admin name though.

**Comment by austintruchan** [comment]
Yay seems to work now!

**Comment by austintruchan** [comment]
oh wait forgot to test form creation

**Comment by austintruchan** [comment]
Oh wait that was already fixed before. Looks good!
